apps/arweave/src/ar_data_sync.erl

```diff
@@ -3008,7 +3056,8 @@ process_disk_pool_matured_chunk_offset(Iterator, TXRoot, TXPath, AbsoluteOffset,
 	%% The other modules will either sync the chunk themselves or copy it over from the
 	%% other module the next time the node is restarted.
 	#sync_data_state{ chunk_data_db = ChunkDataDB, store_id = DefaultStoreID } = State,
-	{Offset, _, ChunkSize, DataRoot, DataPathHash, ChunkDataKey, Key, _StrictSplit} = Args,
+	{Offset, _, ChunkSize, DataRoot, DataPathHash, ChunkDataKey, Key, _BaseSplit, _StrictSplit,
```

Comment:
what do you think about renaming `_BaseSplit, _StrictSplit, _RebaseSupportSplit` to `_PassedBase, _PassedStrictValidation, _PassedRebaseValidation`? I think they are the same values, right? It might make it easier to track the values across function calls / messages.